stdlib/public/Distributed/DistributedActor.swift

```diff
@@ -373,6 +373,28 @@ extension DistributedActor {
   }
 }
 
+/// Supports the operation to produce an any Actor instance from a local
+/// distributed actor
+@available(SwiftStdlib 5.7, *)
+extension DistributedActor {
+  @_alwaysEmitIntoClient
+  @_implements(Actor, unownedExecutor)
+  public nonisolated var __actorUnownedExecutor: UnownedSerialExecutor {
+    if #available(macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0, *) {
+      return unownedExecutor
+    } else {
+      fatalError("FIXME: Implement default actor executor on older platforms")
+    }
+  }
+
+  /// For a local distributed actor, produce an instance of the underlying
+  /// \c Actor.
+  @backDeployed(before: SwiftStdlib 5.11)
+  public var asLocalActor: any Actor {
```

Comment:
Thanks! The `asLocal...` sounds good -- just "asActor" would have been weird, LGTM!